Code/Framework/AzFramework/Platform/Windows/AzFramework/Device/DeviceAttributesCommon_Win.cpp

```diff
@@ -10,37 +10,130 @@
 #include <AzCore/std/string/conversions.h>
 #include <AzFramework/Device/DeviceAttributeDeviceModel.h>
 #include <AzFramework/Device/DeviceAttributeRAM.h>
-#include <AzCore/Console/ILogger.h>
+
+#include <Wbemidl.h>
+#pragma comment(lib, "wbemuuid.lib")
 
 namespace AzFramework
 {
-    DeviceAttributeDeviceModel::DeviceAttributeDeviceModel()
+    AZStd::string QueryWMIProperty(IWbemServices* services, const AZStd::wstring& propertyName)

```

Comment:
This can accept an AZStd::string_view (non-wide character version as a parameter)
```suggestion
    AZStd::string QueryWMIProperty(IWbemServices* services, AZStd::string_view propertyName)

```